src/Backend/Snowflake/Importer.php

```diff
@@ -57,12 +57,12 @@ public function importTable(
         $adapter = $this->getAdapter($source, $destination);
 
         $this->importState = new ImportState(BackendHelper::generateStagingTableName());
-        $this->validateColumns($options, $destination);
+        $this->validateColumns($destination);
 
         $this->runQuery($this->sqlBuilder->getCreateStagingTableCommand(
             $destination->getSchema(),
             $this->importState->getStagingTableName(),
-            $options->getColumns()
+            $destination->getColumnsNames()
```

Comment:
Myslím si že ty columns by měly být u `Source` protože to udává pořadí sloupců ve zdroji (např. CSV) které může být jiné od pořadí sloupců v destination a podle toho se vytváří i staging tabulka.

Reply:
Jo to asi dává smysl aby to byly columns toho CSV.

Reply:
Upravil sem teď má `columnsNames` `SourceInterface`

Reply:
super, jdu to kouknout